core/src/test/scala/org/apache/spark/metrics/InputOutputMetricsSuite.scala

```diff
@@ -198,8 +198,12 @@ class InputOutputMetricsSuite extends SparkFunSuite with SharedSparkContext
     // write files to disk so we can read them later.
     sc.parallelize(cartVector).saveAsTextFile(cartFilePath)
     val aRdd = sc.textFile(cartFilePath, numPartitions)
+    aRdd.cache()
+    aRdd.count()
 
     val tmpRdd = sc.textFile(tmpFilePath, numPartitions)
+    tmpRdd.cache()
+    tmpRdd.count()
```

Comment:
Because we cache the rdd in the CartesianRDD compute method, so there we should count the bytes read from memory.